src/styles/globals.scss

```diff
@@ -120,6 +120,6 @@ $grid-breakpoints: (
 @import 'templates/PostTemplate';
 
 /* ==================== Custom Partials ==================== */
-@import 'carousels';
+@import 'bootstrap';
```

Comment:
I don't think we should import every bootstrap partial as this will negatively affect our bundle size. I think we should only import what we need. Are you looking to import a specific set of Bootstrap styles?

Reply:
This is just our bootstrap overrides file, I renamed `carousels` => `bootstrap` to combine b/c I'm editing parts of tooltip & accordion in bootstrap. Want me to rename this to be more clear? It's not from their library, it's custom overrides

Reply:
Oh, that's my fault; totally missed that! Yes please, could you rename it?